nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -2434,82 +2538,82 @@ Before you begin, confirm that:
 +
 |====
 |*Required Information*|*Description*
-|Source ZooKeeper hostname (*sourceHostname*)|The hostname must be one of the hosts running in the ZooKeeper ensemble, which can be found in <NiFi installation dir>/conf/zookeeper.properties.  Any of the hostnames declared in the *server.N* properties can be used.
-|Destination ZooKeeper hostname (*destinationHostname*)|The hostname must be one of the hosts running in the ZooKeeper ensemble, which can be found in <NiFi installation dir>/conf/zookeeper.properties.  Any of the hostnames declared in the *server.N* properties can be used.
-|Source ZooKeeper port (*sourceClientPort*)|This can be found in *zookeeper.properties* of the <NiFi installation dir>/conf/zookeeper.properties.  The port is specified in the *clientPort* property.
-|Destination ZooKeeper port (*destinationClientPort*)|This can be found in *zookeeper.properties* of the <NiFi installation dir>/conf/zookeeper.properties.  The port is specified in the *clientPort* property.
+|Source ZooKeeper hostname (*sourceHostname*)|The hostname must be one of the hosts running in the ZooKeeper ensemble, which can be found in _<NiFi installation dir>/conf/zookeeper.properties_.  Any of the hostnames declared in the *server.N* properties can be used.
```

Comment:
Should all these bold *propertyNames* be `code formatted` instead?

Reply:
I wasn't sure about the intent for the bold.  But, I could see them being code formatted as you suggested.

Reply:
I think they indicate the variable name, so code formatting seems appropriate.

Reply:
Going to also remove the double reference to zookeeper.properties:

This can be found in zookeeper.properties of the <NiFi installation dir>/conf/zookeeper.properties.